packages/orchestrator-ui-components/src/components/WfoForms/formFields/SummaryField.tsx

```diff
@@ -19,6 +19,7 @@ import { FieldProps, connectField, filterDOMProps } from 'uniforms';
 import { EuiFlexItem, EuiFormRow, EuiText } from '@elastic/eui';
 
 import { getStyles } from '@/components/WfoForms/formFields/SummaryFieldStyling';
+import { getCommonFormFieldStyles } from '@/components/WfoForms/formFields/commonStyles';
```

Comment:
Import can be shorter

Reply:
I think we should fix them in all places then, because on all the other FormFields `getCommonFormFieldStyles `long import is used. IMO if I shorten this one it will make it more confusing.